uast/transformer/semantic.go

```diff
@@ -147,25 +147,36 @@ func (c *commentElems) isTab(r rune) bool {
 }
 
 func (c *commentElems) Split(text string) bool {
-	if !strings.HasPrefix(text, c.StartToken) || !strings.HasSuffix(text, c.EndToken) {
+	trimmed := strings.TrimSpace(text)
+	if !strings.HasPrefix(trimmed, c.StartToken) || !strings.HasSuffix(trimmed, c.EndToken) {
 		return false
 	}
 	text = strings.TrimPrefix(text, c.StartToken)
 	text = strings.TrimSuffix(text, c.EndToken)
 
-	// find prefix
+	// find prefix (if not already trimmed)
 	i := strings.IndexFunc(text, func(r rune) bool {
 		return !c.isTab(r)
 	})
-	c.Prefix = text[:i]
-	text = text[i:]
+
+	if i == -1 {
```

Comment:
This might be nicer:
```golang
c.Prefix = ""
if i >= 0 {
  c.Prefix = text[:i]
  // ...
}
```

Reply:
Yeah, but is assigns twice in the most common case and I don't trust Golang optimizer much, anyway I also find it better and I don't think it'll hurt much.

Reply:
Done.